src/rpc/server.cpp

```diff
@@ -24,7 +24,7 @@
 #include <unordered_map>
 
 static CCriticalSection cs_rpcWarmup;
-static bool fRPCRunning = false;
+static std::atomic<bool> fRPCRunning{false};
```

Comment:
This is the only change needed, why the other changes when there's assign and cast operators?